scripts/helpers/wordcount.js

```diff
@@ -0,0 +1,27 @@
+//Inspired by https://github.com/willin/hexo-wordcount
```

Comment:
I've included the helper directly so the users don't need to install any additional plugins. I've also slightly modified it.